person.rb

```diff
@@ -0,0 +1,34 @@
+require './decorator'
+require './rentals'
+
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :age, :rentals, :name
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..1999)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def correct_name
+    @name = name
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age?
+  end
+
+  def add_rentals=(book, date)
+    Rental.new(book, self, date)
+  end
```

Comment:
- [ ] The method `add_rentals=` serves as a setter typically because of the `=` sign. The previous reviewer asked that you fix that by having only `add_rental`.

- [ ] You should also be mindful of the order of the parameters here. The order of the parameters in the Rental constructor is this, `person`, `book`, date. You need to follow the same order here:

  ```
  def add_rental(book, date)
    Rental.new(self, book, date)
  end
  ```